com/truckmuncher/api/trucks.proto

```diff
@@ -30,11 +30,20 @@ service TruckService {
     /**
      * Modify the serving mode for a truck.
      *
-     * This call requires Vendor authorization
+     * This call is an alias for reportServingMode
+     * This call is deprecated and should no longer be used by clients. It will be removed in the near future.
+     * This call requires the user to be logged in
      */
     rpc modifyServingMode (ServingModeRequest) returns (ServingModeResponse);
```

Comment:
They can be combined. If the reporter is the owner of the truck, they would be verified.